FastTimeSeries/src/models/InceptionTime.jl

```diff
@@ -0,0 +1,83 @@
+"""
+    InceptionModule(ni::Int, nf::Int, ks::Int = 40, bottleneck::Bool = true)
+
+TBW
+"""
+function InceptionModule(ni::Int, nf::Int, ks::Int = 40, bottleneck::Bool = true)
+    ks = [ks ÷ (2^i) for i in range(0, stop = 2)]
+    ks = [ks[i] % 2 == 0 ? ks[i] - 1 : ks[i] for i in range(1, stop = 3)]  # ensure odd ks
+    bottleneck = ni > 1 ? bottleneck : false
+
+    bottleneck_block = bottleneck ? Conv1d(ni, nf, 1, bias = false) : identity
+
+    convs_layers =
+        [Conv1d(bottleneck ? nf : ni, nf, ks[i], bias = false) for i in range(1, stop = 3)]
```

Comment:
I think it makes more sense to have `Conv1d` be lowercase `conv1d`, but that's a minor thing.